datasets/um005/um005.py

```diff
@@ -0,0 +1,137 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+UMC005 English-Urdu is a parallel corpus of texts in English and Urdu language with sentence alignments. The corpus can be used for experiments with statistical machine translation.
+
+The texts come from four different sources:
+- Quran
+- Bible
+- Penn Treebank (Wall Street Journal)
+- Emille corpus
+
+We provide the religious texts of Quran and Bible for direct download. Because of licensing reasons, Penn and Emille texts cannot be redistributed freely. However, if you already hold a license for the original corpora, we are able to provide scripts that will recreate our data on your disk. Our modifications include but are not limited to the following:
+
+- Correction of Urdu translations and manual sentence alignment of the Emille texts.
+- Manually corrected sentence alignment of the other corpora.
+- Our data split (training-development-test) so that our published experiments can be reproduced.
+- Tokenization (optional, but needed to reproduce our experiments).
+- Normalization (optional) of e.g. European vs. Urdu numerals, European vs. Urdu punctuation, removal of Urdu diacritics.
+"""
+_HOMEPAGE_URL = "http://ufal.ms.mff.cuni.cz/umc/005-en-ur/"
+_URL = "http://ufal.ms.mff.cuni.cz/umc/005-en-ur/download.php?f=umc005-corpus.zip"
+_CITATION = """\
+@unpublished{JaZeWordOrderIssues2011,
+author      = {Bushra Jawaid and Daniel Zeman},
+title       = {Word-Order Issues in {English}-to-{Urdu} Statistical Machine Translation},
+year        = {2011},
+journal     = {The Prague Bulletin of Mathematical Linguistics},
+number      = {95},
+institution = {Univerzita Karlova},
+address     = {Praha, Czechia},
+issn        = {0032-6585},
+}
+"""
+
+_ALL = "all"
```

Comment:
Do we have to have a config that concatenates all other datasets? Don't we want to let the user do this in case he want's both configs? Not sure what the rule is here

Reply:
It depends on the dataset. If it is a common usage of the dataset to concatenate everything then we should have a config for it. Otherwise we should just keep the separate configs